libs/core/pinsDigital.cpp

```diff
@@ -101,6 +101,19 @@ void onEvent(DigitalInOutPin pin, PinEvent event, Action body) {
     }    
 }
 
+/**
+ * Measure pin capacitance.
+ */
+//%
+int readCapacitance(DigitalInOutPin pin) {
+    // on STM32F4 this requires 2M pulldown
+    // other chips to be tested
+    pin->setDigitalValue(1);
+    pin->getDigitalValue(PullMode::None);
+    sleep_us(50);
```

Comment:
Should this be a bootloader variable?

Reply:
probably, let me actually see if I can do this in TS